qiskit/transpiler/preset_passmanagers/builtin_plugins.py

```diff
@@ -178,28 +179,35 @@ def pass_manager(self, pass_manager_config, optimization_level=None) -> PassMana
             # use rz, sx, x, cx as basis, rely on physical optimziation to fix everything later one
             stdgates = get_standard_gate_name_mapping()
 
-            def _is_one_op_non_discrete(ops):
+            def _is_one_op_non_discrete(target):
                 """Checks if one operation in `ops` is not discrete, i.e. is parameterizable
                 Args:
                     ops (List(Operation)): list of operations to check
                 Returns
                     True if at least one operation in `ops` is not discrete, False otherwise
                 """
```

Comment:
This documentation comment isn't accurate now (though I think it wasn't 100% perfect before either).